README.md

```diff
@@ -1,11 +1,11 @@
 # Step-up Middleware Client Bundle
 [![Build Status](https://travis-ci.org/OpenConext/Stepup-Middleware-clientbundle.svg)](https://travis-ci.org/OpenConext/Stepup-Middleware-clientbundle) [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/OpenConext/Stepup-Middleware-clientbundle/badges/quality-score.png?b=develop)](https://scrutinizer-ci.com/g/OpenConext/Stepup-Middleware-clientbundle/?branch=develop) [![SensioLabsInsight](https://insight.sensiolabs.com/projects/3fef2be8-1351-4fbb-b6e8-00296c1d8d36/mini.png)](https://insight.sensiolabs.com/projects/3fef2be8-1351-4fbb-b6e8-00296c1d8d36)
```

Comment:
You can replace the Travis badges with GithubActions counterparts!

:badger:

Reply:
a sure... I think :-)